zookeeper-metrics-providers/zookeeper-prometheus-metrics/src/main/java/org/apache/zookeeper/metrics/prometheus/VersionInfoExports.java

```diff
@@ -0,0 +1,44 @@
+package org.apache.zookeeper.metrics.prometheus;
```

Comment:
Hey @maoling, maybe apache license is required?

Reply:
Oops, I will fix it tomorrow. A little surprising, let me re-push this patch and check whether the CI or checkstyle could detect this license missing issue.